platform-coop-toolkit/pcc
# add events template page and share buttons

## Description

- New template page events choose by category
- New field on carbon fields
- Share buttons in the single contents
- Display price in single events

## Steps to test

1. Create a page with template Events and choose category;
2. Fill the event
3. Open the event page with template and see "free" if the field price is not filled or the price if it is filled
4. In the bottom of the single page  event will be displayed the share buttons
5. In the right of the single page event will be displayed the price 

**Expected behavior:** The fields will be displayed on the event page in the sidebar.

## Additional information
